docs/0-installation.md

```diff
@@ -48,14 +48,46 @@ The generator adds these core files, among others:
 * `app/assets/stylesheets/active_admin.scss`
 * `config/initializers/active_admin.rb`
 
-Now, migrate and seed your database before starting the server:
+Now, migrate and seed your database:
 
 ```sh
 rails db:migrate
 rails db:seed
-rails server
 ```
 
+These instructions suggest using the `cssbundling-rails` and `importmap-rails`
+gems for a streamlined asset management setup. However, this is not mandatory,
+and you can handle asset bundling in any way that suits your needs, as long as
+it works effectively.
+
+If you haven't installed and configured `cssbundling-rails` and
+`importmap-rails`, please run:
```

Comment:
Yes this is necessary, but thei setup might change. It's easiest to just say install and configure `cssbundling-rails` and `importmap-rails`